### PR TITLE
add: avatar and badge pages

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,
I have added Avatar and Badge page to my component library.
Please review this PR.

### CODE REVIEW

File: pages/alert.html [hunk 1]
```diff
@@ -1,136 +1,151 @@
...
-        <nav></nav>
-        <sidebar class="sidebar scrollbar"></sidebar>
+      <div class="main">
+        <div class="component-heading">
+          <h2>Alerts</h2>
+          <p>
+            Alerts are used to attract user's attention for important
+            information without interrupting the user's flow.
+          </p>
+          <p></p>
```
Comment (Maianki) [inline pages/alert.html]: I think this ```<p>``` tag is unwanted.

**Review by Maianki — APPROVED** [review]
Suggested a minor change. Everything else looks good.

File: pages/avatar.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        <pre class="component-code-snippet">
+          <code class="language-html">
+            <img class="avatar rounded size-lg" src="/images/profile_picture3.jpeg"/>
+
+            <img class="avatar rounded size-md" src="/images/profile_picture2.jpeg"/>
+
+            <img class="avatar rounded size-sm"src="/images/profile_picture4.jpeg"/>
+          </code>
+        </pre>
+
```
Comment (hsnice16) [inline pages/avatar.html:48]: Just a suggestion, as before starting a new component you are using `<h4>` and `<p>`, and then the component. So, you can wrap that whole component in a `<section>` I think that will increase the readability, that this is a new section.

**Review by hsnice16 — APPROVED** [review]

File: pages/alert.html (hunk 1 above)
Comment (ApurvaSawant11) [reply pages/alert.html]: Yes, it is.  Did not see that. Thanks for pointing it out. I have removed it.

**Comment by mikesch803** [comment]
UI is amazing, everything looks good

**Review by mikesch803 — APPROVED** [review]

**Comment by ApurvaSawant11** [comment]
> UI is amazing, everything looks good

Thank you :)

File: pages/avatar.html (hunk 2 above)
Comment (ApurvaSawant11) [reply pages/avatar.html:48]: Hey, I have made the changes. Thanks!

**Pull request merged**